cocos/2d/assembler/sprite/simple.ts

```diff
@@ -48,8 +48,10 @@ for (let i = 0; i < 4; i++) {
 export const simple: IAssembler = {
     createData (sprite: Sprite) {
         const renderData = sprite.requestRenderData();
-        renderData.dataLength = 2;
+        renderData.dataLength = 4;
         renderData.resize(4, 6);
+        renderData.vertexRow = 2;
+        renderData.vertexCol = 2;
```

Comment:
For the current usage of vertexRow and vertexCol, they are all used inside assemblers, I think it can be an assembler level or type specific constant configuration. They shouldn't be added to RenderData object.

Reply:
Such as Tiled, vertexRow and vertexCol cannot be assigned in constructor, since they are assigned in UpdateRenderData. So that we need these value which are from various components instead of some global places.